validate.lic

```diff
@@ -712,6 +712,22 @@ class DRYamlValidator
       warn("#{list} contains the following spell names not found in data/base_spells, #{name_warnings} Check spelling and capitalization.") unless name_warnings.empty?
     end
   end
+  
+  def assert_that_caravan_training_skills_are_valid(settings)
+    return unless settings.caravan_training_skills
```

Comment:
Check if its empty instead of if it exists. Since it is in base-empty, the empty hash will exist.
```
2.0.0-p648 :001 > test = {}
 => {}
2.0.0-p648 :002 > print "hello" if test
hello => nil
2.0.0-p648 :003 > puts "hello" if test
hello
 => nil
2.0.0-p648 :004 > test.empty?
 => true
```

Reply:
Ah-hah, I did not try that after adding that into base-empty.  Thanks!